sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client.hpp

```diff
@@ -176,6 +176,48 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
         std::string const& name,
         Azure::Core::Context const& context = Azure::Core::Context()) const;
 
+    /**
+     * @brief List the certificate contacts for a specified key vault.
+     *
+     * @details The GetContacts operation returns the set of certificate contact
+     * resources in the specified key vault.
+     *
+     * @remark This operation requires the certificates/managecontacts permission.
+     *
+     * @param context The context for the operation can be used for request cancellation.
+     * @return The contacts list for the key vault certificate.
+     */
+    Azure::Response<std::vector<CertificateContact>> GetContacts(
+        Azure::Core::Context const& context = Azure::Core::Context()) const;
+
+    /**
+     * @brief Delete the certificate contacts for a specified key vault.
```

Comment:
I am curious, what is the specified key vault? This method doesn't take any such params.

Reply:
I copied the comments form the swagger file 

"delete": {
        "tags": [
          "Certificates"
        ],
        "operationId": "DeleteCertificateContacts",
        "description": "**Deletes the certificate contacts for a specified key vault certificate. This operation requires the certificates/managecontacts permission.",**
        "summary": "Deletes the certificate contacts for a specified key vault.",